CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ The Singularity Project has been
 and re-branded as Apptainer.
 For older changes see the [archived Singularity change log](https://github.com/apptainer/singularity/blob/release-3.8/CHANGELOG.md).
 
+## Changes Since Last Release
+
+### New features / functionalities
```

Comment:
I'd like to include this in 1.1.0-rc.3 and for non-major releases we don't distinguish between the type of change.  So please delete this line.  When you redo the commit, change the title to refer to the apptainer PR.

For the final 1.1.0 release I will merge the changes in rc.2 and rc.3 into one full release announcement.